### PR TITLE
release preparation for 5.x

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-214.surge.sh

File: antora.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: graphql
 title: Neo4j GraphQL Library
 version: '5'
+display_version: '5-LTS'
```
Comment (rsill-neo4j) [inline antora.yml]:
```suggestion
display_version: '5 (LTS)'
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by recrwplay — APPROVED** [review]
Apologies - my review wasn't submitted.

I was going to suggest something like `5 (LTS)` here but you beat me to it :)

I don't think we're using this format anywhere else yet, but we will do in future. We could use an attribute to mark a component version as LTS and use that but for now this is fine.

**Pull request merged**